ballista/rust/scheduler/src/main.rs

```diff
@@ -62,17 +62,20 @@ async fn start_server(
         BALLISTA_VERSION, addr
     );
 
-    let scheduler_server =
-        SchedulerServer::new(config_backend.clone(), namespace.clone());
     Ok(Server::bind(&addr)
```

Comment:
I'm a bit concerned this code is creating a new server per client. See https://github.com/apache/arrow/pull/9987/files#r624519673